operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -48,11 +53,18 @@ where
         tile_limit,
         conn_closed,
     )
-    .await?;
-
-    let bytes = gdal::vsi::get_vsi_mem_file_bytes_owned(file_path)?;
+    .await?
+    .into_iter()
+    .map(|x| {
+        gdal::vsi::get_vsi_mem_file_bytes_owned(
+            x.params
+                .expect("params is always set in raster_stream_to_geotiff")
+                .file_path,
```

Comment:
Why don't you use `file_path` from line 44 anymore?

Reply:
line 44 is a prefix (the folder where the files go into). Each timerange is a separate file, so it has its own file name.